Tasks/Semin/task1.py

```diff
@@ -0,0 +1,6 @@
+deposit = int(input('enter the deposit amount at BYN: '))
+term = int(input('enter the term of the deposit in months: '))
+percent = int(input('enter annual percentage: '))
```

Comment:
these values should be decimal to prevent additional calculation errors, -1 point for the unwise choice